apps/consent-ui/src/constants/externalPaths.ts

```
@@ -24,3 +24,4 @@ export const API = {
 };
 
 export const OHCRN_HOME_LINK = 'https://ohcrn.ca/';
+export const OHCRN_EMAIL = 'OHCRN@oicr.on.ca';
```

not sure if this email address is the real one, but either way i think it would be good to add that value as well as the `OHCRN_HOME_LINK` in as env vars instead of constants.

---

is it ok to have these values as the defaults, or just use empty strings?

---

for default values here we can add those as empty strings in the AppConfig setup. Atm I don't think we need to throw errors if they're not defined, these aren't intrinsic to running the application. @mistryrn let me know if you differ on this 😛